connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -534,6 +537,121 @@ private void testOneWayReplicationWithOffsetSyncs(int offsetLagMax) throws Inter
         assertMonotonicCheckpoints(backup, PRIMARY_CLUSTER_ALIAS + ".checkpoints.internal");
     }
 
+    @Test
+    public void testReplicationWithoutOffsetSync() throws Exception {
```

Comment:
That's a pretty big test just to check the offset-syncs and checkpoints topics don't get created. Could we simplify it?

Reply:
Simplified this test now